@navikt/core/icons/meta.json

```diff
@@ -2087,6 +2123,42 @@
     "created_at": "2021-01-21T10:16:34.376Z",
     "pageName": "Family"
   },
+  {
+    "name": "SizeLarge56Px",
```

Comment:
🤔

Reply:
Hmmm
![image](https://user-images.githubusercontent.com/1894119/151799026-afa4271b-0acf-4c21-b824-53dc4837a6a5.png)

Reply:
Hørte med Wei. Gjør samme filterering som med "Pictogram" for "Placeholder"